.zappr.yaml

```diff
@@ -1,15 +1,20 @@
 approvals:
-  # PR needs at least 1 approval
-  minimum: 1
+  # PR needs at least 2 approvals
+  minimum: 2
   # approval = comment that matches this regex
   pattern: "^(:\\+1:|👍)$"
   # note that `from` is by default empty,
   # accepting any matching comment as approval
-  from:
-    # commenter must be either one of:
-    orgs:
-      # a member of norris organization
-      # (any org in here counts)
+  groups:
+   # there must be at least a technical approval from a member of team Norris
+   technical:
+    minimum: 1
+    from:
+     orgs:
       - norris
```

Comment:
Good point, I forgot that.

![image](https://cloud.githubusercontent.com/assets/3198244/20177603/ea09a4f8-a74d-11e6-9c30-b1e9224b2855.png)

Moreover, I think I can remove all of us and UX from collaborators now right?